bokeh/io.py

```diff
@@ -346,6 +356,47 @@ def _save_helper(obj, filename, resources, title, validate):
         with io.open(filename, "w", encoding="utf-8") as f:
             f.write(decode_utf8(html))
 
+def savepng(obj, filename=None, state=None, validate=True):
+    if state is None:
+        state = _state
+
+    if filename is None and state.png:
+        filename = state.png['filename']
+
+    if filename is None:
+        raise RuntimeError("savepng() called but no filename was supplied and output_png(...) was never called, nothing saved")
+
+    with _ModelInDocument(obj):
+        if isinstance(obj, Component):
+            doc = obj.document
+        elif isinstance(obj, Document):
+            doc = obj
+        else:
+            raise RuntimeError("Unable to save object of type '%s'" % type(obj))
+
+        if validate:
+            doc.validate()
+
+        from subprocess import Popen, PIPE
+        from os.path import join, dirname
+
+        from .resources import Resources
+
+        resources = Resources(mode="absolute-dev")
+
+        from .embed import _check_plot_objects, _standalone_docs_json_and_render_items
+        from ._json_encoder import serialize_json
+
+        plot_objects = [obj]
+        plot_objects = _check_plot_objects(plot_objects)
+        (docs_json, render_items) = _standalone_docs_json_and_render_items(plot_objects)
+
+        script = join(dirname(dirname(__file__)), "bokehjs", "render.js")
+        cmd = "node %s --filename=%s --log-level=%s" % (script, filename, resources.log_level)
+
+        proc = Popen(cmd, stdin=PIPE, shell=True)
+        proc.communicate(input=serialize_json(docs_json))
```

Comment:
I don't think you need our `serialize_json` or special `_json_encoder` here, because `docs_json` is plain JSON, not "weird JSON with PlotObject and numpy stuff in it". Simply `json.dumps` should work fine unless I'm missing something.